core/templates/dev/head/pages/collection_editor/editor_tab/CollectionNodeCreatorDirective.js

```diff
@@ -73,6 +90,16 @@ oppia.directive('collectionNodeCreator', [function() {
           );
         };
 
+        var cleanExplorationId = function(newExplorationId) {
```

Comment:
What is the reason for this function? any reason you cant use a javascript slice function here? Also just in general this function just makes me uncomfortable, why have it? There was no string manipulation in previous iterations of the file. If this is due to the typeahead module, then might be best to name the function something with regards to that. I.e. Perhaps the user is entering typeaheadExplorationId, and you would be running convertTypeAheadToExplorationId to clean it.